src/Controls/src/Core/Handlers/Items/iOS/StructuredItemsViewController.cs

```diff
@@ -46,6 +46,16 @@ protected override void Dispose(bool disposing)
 					_footerViewFormsElement.MeasureInvalidated -= OnFormsElementMeasureInvalidated;
 				}
 
+				if (_headerUIView is MauiView hv)
+				{
+					hv.LayoutChanged -= HeaderViewLayoutChanged;
+				}
```

Comment:
If we are now watching the actual method that is doing layout, do we still need the original `_footerViewFormsElement.MeasureInvalidated`? The code seems to end up calling the same `HandleFormsElementMeasureInvalidated` method.

Reply:
IS grabbing the element for footer and header not the all view right? some still need for each one .